script/setup

```diff
@@ -18,6 +18,7 @@ if [ -z "$RAILS_ENV" ] && [ -z "$RACK_ENV" ]; then
   # Do things that need to be done to the application to set up for the first
   # time. Or things needed to be run to to reset the application back to first
   # use experience. These things are scoped to the application's domain.
+  :
```

Comment:
It's [null command](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_16), do nothing, like no-op operator.